### PR TITLE
Consider only t5x tests when determining t5x test status

### PR DESCRIPTION
**Pull request by ashors1**
When running the presubmit CI, t5x's `publish-test` step incorrectly considered the outcomes of all tests rather than just the t5x ones. This PR filters out non-t5x tests to fix to fix this issue.

### CODE REVIEW

**Comment by terrykong** [comment]
How about we do what you did for VIT and prefix the t5 tests with `T5` to allow us to easily filter for it?

**Review by yhtang — APPROVED** [review]
LGTM

**Comment by yhtang** [comment]
Is this in a cyclic dependency situation with https://github.com/NVIDIA/JAX-Toolbox/pull/402 in terms passing the presubmit?

File: .github/workflows/_test_t5x.yaml [hunk 1]
```diff
@@ -326,8 +326,8 @@ jobs:
         | --- | --- | --- |
         EOF
         for i in $EXIT_STATUSES; do
-          # Files are named <GHID>-<NAME>/<NAME>-status.json
-          echo "| $(echo $i | cut -d/ -f1 | cut -d- -f2) | $(jq -r .state $i) | $(jq -r .exitcode $i)"
+          # Files are named t5x-<GHID>-<NAME>/<NAME>-status.json
```
Comment (terrykong) [inline .github/workflows/_test_t5x.yaml:329]: Nit: Should this say `T5X`?

Comment (ashors1) [reply .github/workflows/_test_t5x.yaml]: Yep, fixed. Thanks!

**Review by terrykong — APPROVED** [review]

**Pull request merged**